src/config.rs

```diff
@@ -141,6 +144,8 @@ impl Config {
             self.basedir.display()
         );
         for entry in WalkDir::new(&self.basedir)
+            .follow_links(true)
```

Comment:
I can imagine someone wanting to *not* follow links in their base directory for some reason. Can we make this an option in `Config`?  I think `follow_links` or `follow_symlinks` seem like good names. It's fine if it defaults to true.